phoenix-core/src/test/java/org/apache/phoenix/query/BaseTest.java

```diff
@@ -2006,4 +2007,17 @@ public void run() {
         thread.setDaemon(true);
         thread.start();
     }
+
```

Comment:
Please move this to CompatUtil, the HbaseCompatCapabilities is for bool flags only.